packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -588,9 +604,225 @@ export type CleanNotifications = {
   notificationsById: Record<string, InboxNotificationData>;
 };
 
+function createStore_forNotifications() {
```

Comment:
I really like this new approach a lot 👍🏻

Reply:
Yeah, should make the updates much more "local" and thus easier to reason about.